extractors/segmentationextractor.py

```diff
@@ -1,8 +1,11 @@
 from abc import ABC, abstractmethod
+from spikeextractors.baseextractor import BaseExtractor
 import numpy as np
+from typing import Union
+from .extraction_tools import ArrayType, PathType
 
 
-class SegmentationExtractor(ABC):
+class SegmentationExtractor(ABC, BaseExtractor):
```

Comment:
Are we thinking about changing this to RoiExtractor?

Reply:
I think we can call the repo `RoiExtractors` and keep `SegmentationExtractors` (similar to `SortingExtractor`)

Reply:
ah ok